src/Chullo.php

```diff
@@ -310,6 +310,37 @@ protected function prepareUri($uri, $transaction = "") {
         return implode([$base_uri, $transaction, $relative_path], '/');
     }
 
+    /**
+     * Issues a MOVE request to Fedora.
+     *
+     * @param string    $uri            Resource URI
+     * @param array     $destination    Destination URI
+     * @param string    $transaction    Transaction id
+     *
+     * @return string
+     */
+    public function moveResource($uri,
+                                 $destination,
+                                 $transaction = "") {
+        // Ensure uri takes transaction into account.
+        $uri = $this->prepareUri($uri, $transaction);
+        // Create destinsation URI
+        $destination_uri = "Destination: " . $this->prepareUri($destination, $transaction);
```

Comment:
This results in 502.  No need to manually append 'Destination' as a string.  Just use guzzle's options array.  You need to nest the headers in there.  Looks something like this.
```php
        $destination_uri = $this->prepareUri($destination, $transaction);
        // Create destination array
        $options = [
          'headers' => [
            'Destination' => $destination_uri,
            'Overwrite'   => 'T'
          ],
        ];
        $response = $this->client->request(
          'MOVE',
          $uri,
          $options
        );
```